### DISCUSSION TITLE
Add last deployed commit hash environment variable

### DISCUSSION THREAD

**Discussion by ck-euan** [coollabsio/coolify#3655] [Feature Requests]
I am deploying some apps from my monorepo using turborepo.

I'm trying to use [turbo-ignore](https://turbo.build/repo/docs/guides/skipping-tasks#using-turbo-ignore), and it would be useful to know the commit hash of the last successfully deployed commit at build time. This would allow turbo-ignore to properly skip unaffected builds and speed things up massively for monorepos

**Comment by ahmedsweng** [comment]
> I am deploying some apps from my monorepo using turborepo.
> 
> I'm trying to use [turbo-ignore](https://turbo.build/repo/docs/guides/skipping-tasks#using-turbo-ignore), and it would be useful to know the commit hash of the last successfully deployed commit at build time. This would allow turbo-ignore to properly skip unaffected builds and speed things up massively for monorepos

It is sad that coolify hasn't support for this turbo-ignore feature, I have 3 apps on my server and when I do a push to main branch all of the apps entered the queue to build their container and this crashed my server, even I am only change one project
